packages/protocol/test/common/migration.ts

```diff
@@ -4,35 +4,52 @@ import {
   assertRegistryAddressesSet,
 } from '@celo/protocol/lib/test-utils'
 import { getDeployedProxiedContract } from '@celo/protocol/lib/web3-utils'
+import { ArtifactsSingleton } from '../../migrations/singletonArtifacts'
 
-const getContract = async (contractName: string, type: string) => {
-  if (type === 'proxiedContract') {
-    return getDeployedProxiedContract(contractName, artifacts)
+const getProxiedContract = async (contractName: string, path: string) => {
```

Comment:
I guess this e.g. could be the wrapper that could be exported from somewhere and used everywhere where this pattern is used.